internal/api/v1/admin/contacts/admin_contact_filter_query_builder.go

```diff
@@ -0,0 +1,28 @@
+package contacts
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/bitcoin-sv/spv-wallet-go-client/internal/api/v1/querybuilders"
+	"github.com/bitcoin-sv/spv-wallet-go-client/internal/api/v1/user/contacts"
+	"github.com/bitcoin-sv/spv-wallet/models/filter"
+)
+
+type adminContactFilterBuilder struct {
+	contactFilter filter.AdminContactFilter
+}
+
+func (p *adminContactFilterBuilder) Build() (url.Values, error) {
+	builder := contacts.ContactFilterQueryBuilder{
+		ContactFilter:      p.contactFilter.ContactFilter,
+		ModelFilterBuilder: querybuilders.ModelFilterBuilder{ModelFilter: p.contactFilter.ModelFilter},
+	}
+	params, err := builder.BuildExtendedURLValues()
```

Comment:
`builder.BuildExtendedURLValues()` can return `pram == nil` ?

Reply:
Even if the caller provides an empty, the returned parameters won’t be nil because they are initialized by _NewExtendedURLValues_ within the query builders